generator/shared.go

```diff
@@ -580,6 +588,9 @@ type GenOpts struct {
 	CompatibilityMode      string
 	ExistingModels         string
 	Copyright              string
+
+	Strict          bool
```

Comment:
GenOpts are available from the template, so there is no need to duplicate templates